coral-service/src/main/java/com/linkedin/coral/coralservice/utils/VisualizationUtils.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2022-2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.coralservice.utils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.UUID;
+
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.sql.SqlNode;
+
+import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
+import com.linkedin.coral.incremental.RelNodeIncrementalTransformer;
+import com.linkedin.coral.transformers.CoralRelToSqlNodeConverter;
+import com.linkedin.coral.trino.trino2rel.TrinoToRelConverter;
+import com.linkedin.coral.vis.VisualizationUtil;
+
+import static com.linkedin.coral.coralservice.utils.CoralProvider.*;
+
+
+public class VisualizationUtils {
+
+  public static File getImageDir() {
+    return new File(System.getProperty("java.io.tmpdir") + "/images" + UUID.randomUUID());
+  }
+
+  public boolean isValidFromLanguage(String fromLanguage) {
```

Comment:
Should we change all references of `fromLanguage` to `sourceLanguage`? Similar for `toLanguage` vs `targetLanguage`? This applies to existing APIs.